pkg/rngd/cmd/rngd/rng_linux_arm64.go

```diff
@@ -7,15 +7,47 @@ package main
 import "C"
 
 import (
-	"errors"
+	"encoding/binary"
+	"log"
+	"os"
 )
 
-// No standard RNG on arm64
-
+// Check if there is a hardware RNG device on AArch64, will use that
+// device as the RNG entropy source if we do have
 func initRand() bool {
-	return false
+
+	f, err := os.Open("/dev/hwrng")
+	defer f.Close()
+	if err != nil {
+		log.Fatalf("Cannot open /dev/hwrng: %v", err)
```

Comment:
`log.Fatalf` will exit so the return below will not happen and `rngd` will not run. What is the intention here? Exit or return?

Reply:
Will return false indicate `rngd` will not be launched with `log.Fatalf` removed.